script.js

```diff
@@ -0,0 +1,25 @@
+const passwordBox = document.getElementById("password");
+const length = 15;
+const upperCase = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+const lowerCase = "abcdefhijklmnopqrstuvwxyz";
+const number = "0123456789";
+const symbol = "!@#$%^&*()_+~";
+const allChars = upperCase + lowerCase + number + symbol;
+
+function createPassword(){
+    let password = "";
+    password += upperCase[Math.floor(Math.random() * upperCase.length)];
+    password += lowerCase[Math.floor(Math.random() * lowerCase.length)];
+    password += number[Math.floor(Math.random() * number.length)];
+    password += symbol[Math.floor(Math.random() * symbol.length)];
+
+    while(length > password.length) {
+        password += allChars[Math.floor(Math.random() * allChars.length)];
+    }
+    passwordBox.value = password;
+}
+
+function copyPassword(){
```

Comment:
It is event handler... so rename it to `handleOnClickCopyPassword`

Reply:
Okay done.